pkg/util/selector_test.go

```diff
@@ -310,7 +312,7 @@ func TestClusterMatches(t *testing.T) {
 		want bool
 	}{
 		{
-			name: "cluster excluded",
+			name: "[case 1] cluster excluded",
 			args: args{
 				affinity: policyv1alpha1.ClusterAffinity{
 					ExcludeClusters: []string{cluster.Name},
```

Comment:
Only one `excludeClusters` can't demonstrate it works as expected, we can add `clusterNames` as well,
```golang
				affinity: policyv1alpha1.ClusterAffinity{
					ClusterNames:    []string{cluster.Name},
					ExcludeClusters: []string{cluster.Name},
				},
```
That means even though the cluster is selected by `cluster names` but it can still be filed out by `exclude clusters`.

Reply:
added